src/main/java/se/nackademin/Personell.java

```diff
@@ -0,0 +1,49 @@
+package se.nackademin;
+
+class Personell {
+	private String name;
+	private int age;
+	private String title;
+	private int salary; // per annum
+	private int balance;
```

Comment:
These private instance variables don't have default values, aren't referenced in a constructor and have no setters, so they can never be set to anything.